include/mbedtls/bignum.h

```diff
@@ -186,6 +186,15 @@ typedef uint64_t mbedtls_t_udbl;
     #endif /* !MBEDTLS_NO_UDBL_DIVISION */
 #endif /* !MBEDTLS_HAVE_INT64 */
 
+/*
+ * Sanity check that exactly one of MBEDTLS_HAVE_INT32 or MBEDTLS_HAVE_INT64 is defined,
+ * so that code elsewhere doesn't have to check.
+ */
+#if (!(defined(MBEDTLS_HAVE_INT32) || defined(MBEDTLS_HAVE_INT64))) || \
```

Comment:
This is superfluous: the code immediately above guarantees that at least one of these two macros is set. It makes sense to have a check like this in other files that we might forget to change if we change the code above, but not here.

I don't mind much, so I'm not making this a blocker.